kind/local-env.sh

```diff
@@ -1,44 +1,118 @@
+  echo "Using version ${VERSION}"
+  # need to align with netbox-chart otherwise the creation of the cluster will hang
+  declare -a Remote_Images=( \
+  "gcr.io/kubebuilder/kube-rbac-proxy:v0.14.1" \
+  "busybox:1.36.1" \
+  "docker.io/bitnami/redis:7.2.4-debian-12-r9" \
+  "docker.io/netboxcommunity/netbox:v3.7.8" \
+  "ghcr.io/zalando/postgres-operator:v1.12.2" \
+  "ghcr.io/zalando/spilo-16:3.2-p3" \
+  )
+  NETBOX_HELM_CHART="https://github.com/netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta5/netbox-5.0.0-beta5.tgz"
+
+  # patch load-data.sh
+  sed 's/netbox-demo-v4.1.sql/netbox-demo-v3.7.sql/g' $(dirname "$0")/load-data-job/load-data.orig.sh > $(dirname "$0")/load-data-job/load-data.sh && chmod +x $(dirname "$0")/load-data-job/load-data.sh
+
+  # patch dockerfile (See README at https://github.com/netbox-community/pynetbox for the supported version matrix)
+  sed 's/RUN pip install -Iv pynetbox==7.4.1/RUN pip install -Iv pynetbox==7.3.4/g' $(dirname "$0")/load-data-job/dockerfile.orig > $(dirname "$0")/load-data-job/dockerfile
+elif [[ "${VERSION}" == "4.0.11" ]] ;then
+  echo "Using version ${VERSION}"
+  # need to align with netbox-chart otherwise the creation of the cluster will hang
+  declare -a Remote_Images=( \
+  "gcr.io/kubebuilder/kube-rbac-proxy:v0.14.1" \
+  "busybox:1.36.1" \
+  "docker.io/bitnami/redis:7.4.0-debian-12-r2" \
+  "ghcr.io/netbox-community/netbox:v4.0.11" \
+  "ghcr.io/zalando/postgres-operator:v1.12.2" \
+  "ghcr.io/zalando/spilo-16:3.2-p3" \
+  )
+  NETBOX_HELM_CHART="https://github.com/netbox-community/netbox-chart/releases/download/netbox-5.0.0-beta.84/netbox-5.0.0-beta.84.tgz"
+  
+  # patch load-data.sh
+  sed 's/netbox-demo-v4.1.sql/netbox-demo-v4.0.sql/g' $(dirname "$0")/load-data-job/load-data.orig.sh > $(dirname "$0")/load-data-job/load-data.sh && chmod +x $(dirname "$0")/load-data-job/load-data.sh
+  
+  cp $(dirname "$0")/load-data-job/dockerfile.orig $(dirname "$0")/load-data-job/dockerfile
+elif [[ "${VERSION}" == "4.1.8" ]] ;then
+  echo "Using version ${VERSION}"
+  # need to align with netbox-chart otherwise the creation of the cluster will hang
+  declare -a Remote_Images=( \
+  "gcr.io/kubebuilder/kube-rbac-proxy:v0.14.1" \
+  "busybox:1.37.0" \
+  "docker.io/bitnami/redis:7.4.1-debian-12-r2" \
+  "ghcr.io/netbox-community/netbox:v4.1.8" \
+  "ghcr.io/zalando/postgres-operator:v1.12.2" \
+  "ghcr.io/zalando/spilo-16:3.2-p3" \
+  )
+
+  # create load-data.sh
+  cp $(dirname "$0")/load-data-job/load-data.orig.sh $(dirname "$0")/load-data-job/load-data.sh
+
+  cp $(dirname "$0")/load-data-job/dockerfile.orig $(dirname "$0")/load-data-job/dockerfile
+else
+  echo "Unknown version ${VERSION}"
+  exit 1
 fi
 
+for img in "${Remote_Images[@]}"; do
+  docker pull "$img"
+  kind load docker-image "$img"
+done
+
 # build image for loading local data via NetBox API
 cd ./kind/load-data-job && docker build -t netbox-load-local-data:1.0 --no-cache --progress=plain -f ./dockerfile . && cd -
```

Comment:
On my macOS Podman 5.2.4, this will build but not tag the image when running `make create-kind`:

```bash
...
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load

What's Next?
  View summary of image vulnerabilities and recommendations → docker scout quickview
/tmp/netbox-operator
using podman due to KIND_EXPERIMENTAL_PROVIDER
enabling experimental podman provider
ERROR: image: "netbox-load-local-data:1.0" not present locally
make: *** [create-kind] Error 1
```

`docker image ls` will not show the image then. If I swap from the `docker` cli to the `podman` cli, it works.

I think adding the `--load` parameter should fix this for podman:

```suggestion
cd ./kind/load-data-job && docker build -t netbox-load-local-data:1.0 --load --no-cache --progress=plain -f ./dockerfile . && cd -
```

Reply:
Testing CI now

Reply:
Passed.